jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/html/AbstractHtmlContainerLoader.java

```diff
@@ -40,5 +49,25 @@ public void loadComponent() {
         componentLoader().loadWhiteSpace(resultComponent, element);
         componentLoader().loadSizeAttributes(resultComponent, element);
         componentLoader().loadThemeList(resultComponent, element);
+
+        String property = element.attributeValue("property");
+        if (!Strings.isNullOrEmpty(property) && resultComponent != null) {
+            Optional<InstanceContainer<?>> instanceContainer = getDataLoaderSupport().loadContainer(element, property);
+            instanceContainer.ifPresent(container -> getHtmlDataBinding().bind(resultComponent, container, property));
+        }
```

Comment:
1. I'm sure we should move this into a separate `loadData(resultComponent, element)` method.
2. Unnecessary variable `instanceContainer` it seems:

```suggestion
getDataLoaderSupport().loadContainer(element, property)
            .ifPresent(container -> getHtmlDataBinding().bind(resultComponent, container, property));
```

3. Result component can be null? WAT? 54 line